src/commands/hardis/lint/access.ts

```diff
@@ -57,40 +57,40 @@ export default class Access extends SfdxCommand {
   protected static sourceElements = [
     {
       regex: `/**/*.cls`,
-      type : 'Apex classes',
```

Comment:
@mickaelgudin please use type names of package.xml: ApexClass, CustomField, CustomObject... :)